submissions/5mountains/tiny-js-world/index.js

```diff
@@ -0,0 +1,96 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const man = {},
+   woman = {},
+   dog = {},
+   cat = {},
+   catWoman = {};
+
+man.species ='human';
+man.name = 'Tom';
+man.gender = 'male';
+man.legs = 2;
+man.hands = 2;
+man.paws = 0;
+man.saying = 'Hello Jenny!';
+man.friends = ['Tom', 'Jenny', 'Rex', 'Selina'];
```

Comment:
The above fragment might look like the following:
```
const man = {
    species: 'human',
    name: 'Tom',
    gender: 'male',
    // ... etc
  },
  woman = {
    species: 'human',
    // ... etc
  },
  // ... etc
```

Reply:
done